### PR TITLE
add Increment example

### PR DESCRIPTION
**Pull request by A-Iskakov**
Increment example with `google-cloud-firestore >= 1.2.0`

### CODE REVIEW

**Comment by gguuss** [comment]
@A-Iskakov You need to add your email and GitHub information [as described in Contributing](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/CONTRIBUTING.md) and then reply to the PR. If you have already verified your email, I can manually verify for you.

File: firestore/cloud-client/snippets.py [hunk 1]
```diff
@@ -384,6 +384,15 @@ def update_server_timestamp():
     })
     # [END update_server_timestamp]
 
+    
+def update_data_increment():
+    db = firestore.Client()
```
Comment (gguuss) [inline firestore/cloud-client/snippets.py]: Is there any reason this is outside of the snippet?

File: firestore/cloud-client/snippets.py [hunk 2]
```diff
@@ -384,6 +384,15 @@ def update_server_timestamp():
     })
     # [END update_server_timestamp]
 
+
```
Comment (gguuss) [inline firestore/cloud-client/snippets.py]: Trim the whitespace on this line

File: firestore/cloud-client/snippets.py [hunk 3]
```diff
@@ -384,6 +384,15 @@ def update_server_timestamp():
...
 
+    
+def update_data_increment():
+    db = firestore.Client()
+    # [START update_data_increment]
+    city_ref = db.collection(u'cities').document(u'SF')
+    city_ref.update({
+        u'population': Increment(1)
+    })
+    # [END update_data_increment]
```
Comment (gguuss) [inline firestore/cloud-client/snippets.py]: Add another newline here

**Comment by andrewferlitsch** [comment]
@gguuss Looks like submitter never came back and sign CLA. Should we close this?

**Pull request closed without merging**